plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/actions/AFeatureModelAction.java

```diff
@@ -64,4 +67,25 @@ private boolean isExternalFeature(IFeature feature) {
 		return (feature != null) && (feature instanceof MultiFeature) && ((MultiFeature) feature).isFromExtern();
 	}
 
+	/**
+	 * method to check if the selection in the editor includes a feature from an external submodel
+	 *
+	 * @param selection the selection from the editor
+	 *
+	 * @return true if there is a feature from an external submodel, false otherwise
+	 */
+	protected boolean hasExternalFeature(IStructuredSelection selection) {
+		for (final Object selectedElement : selection) {
```

Comment:
This crashes the CI as it does not compile / work under Java 8. Please fix asap! @Rahel12342